app/src/main/java/com/example/database/room/PlayerViewModel.kt

```diff
@@ -0,0 +1,17 @@
+package com.example.database.room
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.ViewModel
+
+class PlayerViewModel(
+    private val playerDao: PlayerDao,
+) : ViewModel() {
+
+    val liveData: LiveData<List<Player>> = playerDao.observePlayers()
```

Comment:
лучше по названиям писать что ожидается на выход, так не понятно без контекста
```suggestion
    val playersData: LiveData<List<Player>> = playerDao.observePlayers()
```